src/routes/beast.ts

```diff
@@ -1,11 +1,29 @@
 import express from 'express';
 
 import BeastController from '../controller/beast';
+import { validateId, validationForCreateBeast, validationForUpdateBeast } from '../validators/beastValidators';
+import { validate } from '../validators/validator';
 
 const router = express.Router();
 
-router.post('/', async (req, res) => BeastController.create(req, res));
-router.get('/:id', async (req, res) => BeastController.getBeastById(req, res));
-router.patch('/:id', async (req, res) => BeastController.updateBeast(req, res));
+router.post(
+    '/',
+    validationForCreateBeast(),
+    validate,
+    async (req: express.Request, res: express.Response) => BeastController.create(req, res),
+);
+router.get(
+    '/:id',
+    validateId(),
+    validate,
+    async (req: express.Request, res: express.Response) => BeastController.getBeastById(req, res),
```

Comment:
Can we import Request and Response from express rather than express.Request?

Reply:
Yea was having some weird issue where it wouldnt recognize my import but will try again and see if it works